File: Medium/Longest_Consecutive_Sequence/Longest_Consecutive_Sequence.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    longestConsecutive(nums) {

        let mySet = new Set(nums);
        let res = 0;

        for (let num of mySet) {
            if (!mySet.has(num - 1)) {
                let l = 1;
                while (mySet.has(num + l)) {
                    l++;
                }
                res = Math.max(res, l);
            }
        }

        return res;
    }
}

File: Medium/Longest_Consecutive_Sequence/Longest_Consecutive_Sequence.java
            mySet.add(num);
        }

        for (int num : mySet) {
            if (!mySet.contains(num - 1)) {
        }
                int l = 1;
            }
                while (mySet.contains(num + l)) {
                    l++;
                }
                res = Math.max(res, l);
        int res = 0;

        return res;
    }
}


